sos/plugins/openstack_cinder.py

```diff
@@ -37,14 +37,23 @@ def setup(self):
             "systemctl status openstack-cinder-api.service"
         )
 
+        ps_status = self.get_command_output("ps -ef")
+        in_ps = False
+        if ps_status['status'] == 0:
+            for line in ps_status['output'].splitlines():
+                if "cinder_wsgi" in line:
+                    in_ps = True
+                    break
+
         container_status = self.get_command_output("docker ps")
         in_container = False
         if container_status['status'] == 0:
             for line in container_status['output'].splitlines():
                 if line.endswith("cinder_api"):
                     in_container = True
+                    break
 
-        if (service_status['status'] == 0) or in_container:
+        if (service_status['status'] == 0) or in_container or in_ps:
```

Comment:
Do we really need three separate conditions to enable this? And why is there a test for `in_container` inside the block? This does not seem all that readable or optimal right now.

Why not place the set up of `cinder_config` outside this block, simplifying it and placing the `cinder-manage` command collection closer to the condition that enables it?

I suspect that would also allow the condition to be simplified.

Reply:
I reviewed that part and removed one condition by checking the ps for either cinder-api or cinder_wsgi. with this we can remove one of the conditions. 

Also moved the cinder_config part in when we check for running cinder in a container env.